assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -57,6 +72,17 @@ function MyStories() {
     fetchStories({ status });
   }, [fetchStories, status]);
 
+  const filteredStories = useMemo(() => {
+    return stories.filter((story) => {
+      const lowerTypeaheadValue = typeaheadValue.toString().toLowerCase();
```

Comment:
good point, thanks - i was tracking down the value being a number because of the id but now that's getting updated separately and irrelevant to the typeahead value.

Reply:
oh wait i remember why i did this! 
So yes actually this is important at the moment. Right now if you're just typing into the input you're getting a value of a string BUT if you select an item from the dropdown you're getting that item's _value_ which is separate from the label - because when I'd originally designed the component I'd assumed we would want unique items (like a 1:1) so it's actually the ID of the story which is a number. 

the more info I'm getting about how this search is supposed to work the more I feel like the label should just be the formatted version of the value, no id, unless tags and categories have IDs associated with them that we're searching on. 

Anyway, this is why I'd left the toString here.

Reply:
Ahhh, yes, I understand, that makes sense 🤔

Reply:
Could it be `toString().trim()`?  When I type in a letter and a space, I'm getting no matches but with trim, it would match.